app/client/cli/account.go

```diff
@@ -0,0 +1,72 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/pokt-network/pocket/shared/crypto"
+	"github.com/pokt-network/pocket/utility/types"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	actorCmd := NewAccountCommand()
```

Comment:
`actorCmd` should be `accountCmd`

Reply:
https://github.com/pokt-network/pocket/pull/177/commits/ce28724632ec7e647dcc04da62aa24565ef449fb